### ISSUE TITLE
Module loading does not working after last merge

### ISSUE THREAD

**Issue by 0xf104a** [Andrewerr/ddos2#7]
**Describe the bug**

**To Reproduce**
1. Build program
2. Run `./ddos2`

**Expected behavior**
Loaded modules

**Desktop (please complete the following information):**
 - OS: Arch Linux 5.6.14-arch1-1
 - Compiler: GCC 10.1.0

**Comment by 0xf104a** [comment]
The names from libddos2 are undefined
`nm` output:
```
00011e78 a _DYNAMIC
00011f60 a _GLOBAL_OFFSET_TABLE_                                                           00000b48 t __atexit_handler_wrapper                                                        00012014 B __bss_start
         U __cxa_atexit@@LIBC
         U __cxa_finalize@@LIBC
00012000 d __dso_handle
00000b40 t __emutls_unregister_key
00000b30 t __on_dlclose
00000b44 t __on_dlclose_late
         U __register_atfork@@LIBC
00012014 D _edata
0001201c B _end
         U argument_add
         U argument_value_get
         U array_add
         U array_create
00000b54 t atexit
00012018 B conn
         U connection_open
         U ddos2_begin
         U ddos2_modconfig
         U die
         U fflush@@LIBC
         U free@@LIBC
         U get_iface
         U info
         U malloc@@LIBC
00012008 D mod_author
00012010 D mod_description
00012004 D mod_name
00000c98 T mod_on_init                                                                     00000b84 T mod_on_load
00000c38 T mod_on_run                                                                      0001200c D mod_version
00001648 r ndk_build_number                                                                00001608 r ndk_version
000015f0 r note_android_ident
00001604 r note_data
00001688 r note_end                                                                        000015fc r note_name
         U packet_create
         U printf@@LIBC                                                                    00000b74 t pthread_atfork
         U stdout@@LIBC                                                                             U strlen@@LIBC
         U success                                                                         00000cc4 T test_add
00000d60 T test_all                                                                        000011c0 T test_close
00000f2c T test_init                                                                       000010e8 T test_long_send
00000f84 T test_open
0000100c T test_send                                                                       00012014 B tests
00000c9c T tests_begin
00001218 T udp_tests_prepare
00001334 T udp_tests_register
```

**Comment by 0xf104a** [comment]
Fixes were merged with pull request #8
